src/core/mount.c

```diff
@@ -1017,8 +1017,11 @@ static void mount_enter_mounting(Mount *m) {
 
         /* Create the source directory for bind-mounts if needed */
         p = get_mount_parameters_fragment(m);
-        if (p && mount_is_bind(p))
-                (void) mkdir_p_label(p->what, m->directory_mode);
+        if (p && mount_is_bind(p)) {
+                r = mkdir_p_label(p->what, m->directory_mode);
+                if (r < 0)
+                        log_unit_error_errno(UNIT(m), r, "Failed to make bind mount source '%s': %m", p->what);
```

Comment:
this should be log_unit_warning_errno(), i.e. downgrade the log level here.

Our general rule is: use LOG_ERR when we actually fail something and LOG_WARNING when we proceed ignoring the failure. Since the code continues here this hence should be LOG_WARNING

Reply:
Given that the subsequent mount would fail, doesn't this meet the criteria for LOG_ERR?